client/tokenized-express-checkout/blocks/components/express-checkout-component.js

```diff
@@ -95,10 +95,10 @@ const ExpressCheckoutComponent = ( {
 	} );
 	const onClickHandler = ! isPreview ? onButtonClick : () => {};
 	const onShippingAddressChange = ( event ) =>
-		shippingAddressChangeHandler( api, event, elements );
```

Comment:
The `api` argument is no longer needed when changing the shipping address or selecting a shipping rate.